socorro/unittest/external/boto/test_crash_data.py

```diff
@@ -35,14 +34,13 @@ def test_get_processed(self, boto_helper):
 
     @mock_s3_deprecated
     def test_get_processed_not_found(self, boto_helper):
-        with mock.patch("socorro.lib.transaction.BACKOFF_TIMES", [0]):
-            boto_helper.get_or_create_bucket("crashstats")
+        boto_helper.get_or_create_bucket("crashstats")
 
-            boto_s3_store = self.get_s3_store()
-            with pytest.raises(CrashIDNotFound):
```

Comment:
I need to check if `CrashIDNotFound` will trigger retrying. Seems like this test expects it to.

Reply:
Neither `CrashIDNotFound` nor the two response errors `KeyNotFound` and `boto.exceptions.StorageError` trigger retrying. This test suggests it should have, but I think either the code changed at some point between when this test was written and now or this test was written curiously.

So... to summarize, retrying isn't triggered and I don't think it should at this time, so we're all good here.